tests/test_bucket_operations.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from framework import config
+from framework.connection import SSHConnection
+
+log = logging.getLogger(__name__)
+
+
+def test_bucket_operations(account_manager, bucket_manager, unique_resource_name, random_hex):
+    SSHConnection().connection
+    # Bucket operations
+    account_name = unique_resource_name(prefix="account")
+    access_key = random_hex()
+    secret_key = random_hex()
+    config_root = config.ENV_DATA["config_root"]
+    account_manager.create(account_name, access_key, secret_key, config_root)
+    account_manager.list(config_root)
+    bucket_name = unique_resource_name(prefix="bucket")
+    bucket_manager.createBucket(account_name, bucket_name, config_root)
+    bucket_manager.listBucket()
+    bucket_manager.deleteBucket(bucket_name)
+    account_manager.delete(config_root, account_name)
+    account_manager.list()
```

Comment:
For the sake of readability, you can remove the explicit usage of `config.ENV_DATA["config_root"]` since we're using the `config.ENV_DATA["config_root"]` as the default value for the config_root argument in the account_manager and bucket_manager methods:

```suggestion
def test_bucket_operations(account_manager, bucket_manager, unique_resource_name, random_hex):
    SSHConnection().connection
    # Bucket operations
    account_name = unique_resource_name(prefix="account")
    access_key = random_hex()
    secret_key = random_hex()
    account_manager.create(account_name, access_key, secret_key)
    account_manager.list()
    bucket_name = unique_resource_name(prefix="bucket")
    bucket_manager.createBucket(account_name, bucket_name)
    bucket_manager.listBucket()
    bucket_manager.deleteBucket(bucket_name)
    account_manager.delete(account_name)
    account_manager.list()
```

Reply:
Fixed